framework/src/main/java/org/apache/drill/test/framework/ColumnList.java

```diff
@@ -88,8 +93,36 @@ public int hashCode() {
   public String toString() {
     StringBuilder sb = new StringBuilder();
     for (int i = 0; i < values.size() - 1; i++) {
+      int type = (Integer) (types.get(i));
+      if (categories.contains("smoke_simba_jdbc") && (type == Types.VARCHAR)) {
```

Comment:
Sure.  I was thinking that the Simba tests would be parallel to the smoke tests.